CONTRIBUTING.md

```diff
@@ -437,65 +441,44 @@ The most important parts of this module are:
 1. The `SAMPLE_DATA` download manager object.
 2. The `list_datasets()` function, which returns a list of the available poses and bounding boxes datasets (file names of the data files).
 3. The `fetch_dataset_paths()` function, which returns a dictionary containing local paths to the files associated with a particular sample dataset: `poses` or `bboxes`, `frame`, `video`. If the relevant files are not already cached locally, they will be downloaded.
-4. The `fetch_dataset()` function, which downloads the files associated with a given sample dataset (same as `fetch_dataset_paths()`) and additionally loads the pose or bounding box data into movement, returning an `xarray.Dataset` object. If available, the local paths to the associated video and frame files are stored as dataset attributes, with names `video_path` and `frame_path`, respectively.
+4. The `fetch_dataset()` function, which downloads the files associated with a given sample dataset (same as `fetch_dataset_paths()`) and additionally loads the pose or bounding box data into `movement`, returning an `xarray.Dataset` object. If available, the local paths to the associated video and frame files are stored as dataset attributes, with names `video_path` and `frame_path`, respectively.
```

Comment:
do we want to link the module and these functions to the api ref?

Reply:
btw there's inconsistency throughout the file with monospacing `movement`.

Reply:
Yeah, I noticed that, but I only corrected those in the sample data section. I'm thinking of enforcing the consistency in a separate PR...

Reply:
Sounds good to me! Can be resolved by opening an issue.

Reply:
Already started a PR, which will need to be rebased after this one is merged, see https://github.com/neuroinformatics-unit/movement/pull/626